### PR TITLE
chore(.github/action.yml): enable autopublish

### LINKED ISSUE #23: Enable auto-publish on TR
We need to: 
https://github.com/w3c/echidna/wiki/How-to-use-Echidna-with-ReSpec-and-GitHub

### PR DESCRIPTION
**Pull request by marcoscaceres**
closes #23

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/image-resource/pull/24.html" title="Last updated on May 8, 2020, 8:18 AM UTC (634456e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/image-resource/24/4987d8b...634456e.html" title="Last updated on May 8, 2020, 8:18 AM UTC (634456e)">Diff</a>

### CODE REVIEW

**Comment by marcoscaceres** [comment]
@sidvishnoi, this look ok to you? Do you know if actions work directly off /.github/ or do I need to click things?

**Comment by sidvishnoi** [comment]
This action.yml file needs to be in `.github/workflows/` folder. I'll update the docs.

Also, need to add ECHIDNA_TOKEN in GitHub repo secrets (that's the clicking part), if not done already.

File: .github/action.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      - master
+  pull_request: {}
+
+jobs:
+  validate-and-publish:
+    name: Validate and Publish
+    runs-on: ubuntu-latest # only linux supported at present
+    steps:
+      - uses: actions/checkout@v1
+      - uses: w3c/respec-w3c-auto-publish@master # use the action
```
Comment (sidvishnoi) [inline .github/action.yml:15]: Maybe use `@v1`. Master can have breaking changes. v1 is semver safe and moving release tag.

Comment (marcoscaceres) [reply .github/action.yml]: ok, cool. Might want to update this also in other repo.

**Comment by marcoscaceres** [comment]
> Also, need to add ECHIDNA_TOKEN in GitHub repo secrets (that's the clicking part), if not done already.

Token added! Thanks for the reminder.

**Comment by marcoscaceres** [comment]
Yay, working already :) Complaining about wrong GitHub branch 👍

**Comment by marcoscaceres** [comment]
@rayankans I seem to be missing your W3C ID... if you got to: 
https://www.w3.org/users/myprofile

It will redirect you to a URL that contains your id (`user/21312312` or something).

**Pull request merged**